docs/content/en/docs/crd-ref/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: CRD Reference pages
+description: Reference information about the KLT CRDs
+weight: 100
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+This section provides comprehensive reference information
+about the [Customized Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRDs)
+that are defined for the Keptn Lifecycle Toolkit.
+See [Resource Management for Pods and Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)
+for more information about how resources are used
+to configure Kubernetes pods.
```

Comment:
I'm not sure if this is relevant at this point ...

Reply:
Okay, it's gone.